packages/shared/src/variants/__tests__/super_tic_tac_go.test.ts

```diff
@@ -0,0 +1,42 @@
+import { SuperTicTacGo, Nonant } from "../super_tic_tac_go";
+import { Color } from "../baduk";
+
+const B = Color.BLACK,
+  _ = Color.EMPTY,
+  W = Color.WHITE;
+
+test("Play a game", () => {
+  const game = new SuperTicTacGo({ width: 9, height: 9, komi: 0 });
+  expect(game.nextToPlay()).toEqual([0]);
+  expect(game.getLegalVertices().length).toEqual(81);
+  expect(() => game.getLegalNonant()).toThrow();
```

Comment:
nit: for variant tests, I try not to test internals (meaning that I only test methods of `AbstractGame`)

In this case, we could probably remove all the `.getLegalNonant()` and `.getLegalVertices()` calls, as they are implicitly tested by `game.playMove()`.

Not only does this make future refactors easier, it also simplifies the test.  Then it's also possible to make these methods/classes private.

- https://en.wikipedia.org/wiki/Encapsulation_(computer_programming)
- https://softwareengineering.stackexchange.com/questions/100959/how-do-you-unit-test-private-methods